### DISCUSSION TITLE
desktop accessible app library manager

### DISCUSSION THREAD

**Discussion by asanghc** [ImranR98/Obtainium#1412] [Ideas]
We need a desktop accessible manager that would facilitate the creation and maintaining of multiple obtainium application lists. Think a GUI to paste source URLs and drag and drop between lists, toggle between different sources...
It should be web-based so it can be used from any Desktop (and even mobile) OS.

I'm guessing at least some of the code could be shared from the obtainium app?

I'm constantly importing/exporting and merging json files and losing my mind.
A mobile interface is just not practical for the initial setup.
We do it now, because there just isn't anything better yet.

Am I the only user with this issue?

**Comment by DarkCrypt** [comment]
I've been using Obtainium for a long time and never used the import/export function.

You're asking for a web based GUI to organize json files?

**Reply by asanghc** [reply]
> I've been using Obtainium for a long time and never used the import/export function.
> 
> You're asking for a web based GUI to organize json files?

If you don't have multiple android devices and have never used the import/export function, this would be of little use to you.
Web-based JSON editors are aplenty. I have not found anything that simplifies the task of maintaining several lists of (100s-1000s) apps for a dozen different devices. This isn't my full time gig though...
